service/k8sclient/internal/kubectl/kubectl.go

```diff
@@ -242,10 +242,14 @@ func run(ctx context.Context, kubectlCmd []string, args []string, stdin interfac
 
 	var inBuf bytes.Buffer
 	if stdin != nil {
-		e := json.NewEncoder(&inBuf)
-		e.SetIndent("", "  ")
-		if err := e.Encode(stdin); err != nil {
-			return nil, err
+		if b, ok := stdin.([]byte); ok {
```

Comment:
Do we add this because operators manifests we apply are []byte not string?

Reply:
we add it because it's []byte, not struct.